inception-doc/src/main/resources/META-INF/asciidoc/user-guide/getting-started.adoc

```diff
@@ -1,79 +1,399 @@
-====
+and you find all of these in the project *settings* just mentioned.
+Again, for details on any section/topic mentioned, please check the main
+documentation.
 
-[NOTE]
-====
-By default the server starts on port 8080 and you can access it via a browser at `http://localhost:8080` after you started it. But if you already have a service running on that port, you can add the parameter `-Dserver.port=9999` at the end of the command line to start the server on port 9999 (or choose any other port).
-====
+* In the section **Documents, **you may upload your files to be
+annotated. Make sure that the format selected (see the dropdown on the
+right) is the same as the file.
+* In the section **Users, **you may add users to your project and change
+their rights in this project. You can only add existing users here from
+the dropdown (left).
+** In order to *add new users* so they can be chosen from this dropdown,
+click on „*administration*“- button in the very top right corner and
+select section *Users* on the left. For *user roles* (within the whole
+instance of INCEpTION) see
+https://zoidberg.ukp.informatik.tu-darmstadt.de/jenkins/job/INCEpTION%20(GitHub)%20(master)/de.tudarmstadt.ukp.inception.app$inception-app-webapp/doclinks/1/#sect_users
+** *Selecting a user from the dropdown* in the project settings section
+*Users*, you can check/unchek the *user‘s rights *(they are different
+from roles and count for this project only) on the right side. Any
+combination is possible and the user will always have _all_ rights
+combined.
+
+[cols=",,",]
+|===
+|User right |Description |*Access to Dashborad Sections *
+
+|Annotator |- annotate only a|
+- Annotation
+
+- Knowledge Base
+
+|Courator |- curate only a|
+- Couration
+
+- Monitoring
+
+- Agreement
+
+- Evaluation
+
+|Project Manager |- annotate +
+- courate +
+- create projects +
+- add new documents +
+- add guidelines +
+- manage users +
+- open documents like an other user sees it (read only) |All pages
+available
+|===
+
+* In the section **Layers, **you may modify existing layers to use for
+your annotation and create own layers. In case you do not want to work
+on default layers only, we highly recommend reading the main
+documentation on layers. LINK EINFÜGEN
+* In the section **Knowledge Base, **you can change the settings for the
+knowledge bases used in your project and also import local and remote
+knowledgebases or create a new one. Herefore, just click the
+„create“-button and follow the wizzard.
+** Note that you can have several knowledge bases in your INCEpTION
```

Comment:
I do not understand what you are saying here

Reply:
I don't know what is the problem - but I reformulated to be more clear in structure. does it make sense to you now?